forge-game/src/main/java/forge/game/cost/CostEnlist.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Forge: Play Magic: the Gathering.
+ * Copyright (C) 2011  Forge Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package forge.game.cost;
+
+import java.util.Map;
+
+import forge.game.ability.AbilityKey;
+import forge.game.card.Card;
+import forge.game.player.Player;
+import forge.game.spellability.SpellAbility;
+import forge.game.trigger.TriggerType;
+
+/**
+ * The Class CostExert.
+ */
+public class CostEnlist extends CostPartWithTrigger {
+
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Instantiates a new cost Exert.
+     * 
+     * @param amount
+     *            the amount
+     * @param type
+     *            the type
+     * @param description
+     *            the description
+     */
+    public CostEnlist(final String amount, final String type, final String description) {
+        super(amount, type, description);
+    }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see forge.card.cost.CostPart#toString()
+     */
+    @Override
+    public final String toString() {
+        final StringBuilder sb = new StringBuilder();
+        sb.append("Enlist " + this.getType());
+        return sb.toString();
+    }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see
+     * forge.card.cost.CostPart#canPay(forge.card.spellability.SpellAbility,
+     * forge.Card, forge.Player, forge.card.cost.Cost)
+     */
+    @Override
+    public final boolean canPay(final SpellAbility ability, final Player payer, final boolean effect) {
+        return true;
+    }
+
+    @Override
+    protected Card doPayment(SpellAbility ability, Card targetCard, final boolean effect) {
+        targetCard.tap(true);
+        // need to transfer info
+        for (Card c : getLKIList()) {
+            payTrig.addRemembered(c);
```

Comment:
Should we even take the LKI or just use the targetCard? it's questionable

Reply:
> Enlist counts the power of the tapped creature as the trigger resolves.

meaning we should probably use LKI there, and have the Pump effect get the real value?

Reply:
I checked, `TriggerRemembered$CardPower` does give us the LKICopy e.g. last battlefield state